Linear_Regression_Using_Straight_Line/Linear_Regression_using_Straight_Line.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
```

Comment:
Can you please remove this Linear_Regression_using_Straight_Line.py from this PR